public/js/splitjs/splitjs.directive.js

```diff
@@ -0,0 +1,20 @@
+'use strict'; 
+
+module.exports =
+angular
+.module('diSplitJs.directive', [])
+.directive('split-area'), function(){
```

Comment:
I think you already found this, but I wanted to flag it since it took me a bit to find too. Should be 
```
.directive('split-area', function(){
```